README.md

```diff
@@ -556,6 +556,17 @@ Example 4: After P-1♣️!-1♥️-1♠️-3♣️!, note that the initial pass
           deny the asked queen but show another queen.
     * DOPI over interference (Double = 0 or 3, Pass = 1 or 4, Step = 2 or 5 w/o
       queen, 2 steps = 2 or 5 w/ queen, assuming space permits)
+* Exclusion Keycard
+    * Jump to 5 of a suit shows a void in that suit and asks for keycards
```

Comment:
Is this after we establish a fit?

Reply:
It's like keycard in that it implicitly establishes the trump suit if needed.